app/[lang]/not-found.tsx

```diff
@@ -0,0 +1,37 @@
+import '../../styles/globals.css'
+import { Button } from 'shared'
+import Topbar from 'components/Topbar'
+import Image from 'next/image'
+
+export default function NotFound() {
+  return (
+    <div className="flex h-full grow items-end justify-center">
+      <Image
+        src="/assets/images/404.jpg"
+        alt="page not found"
+        fill
+        quality={100}
+        loading="eager"
+        priority
+        className="-z-20 h-full w-full object-cover"
+      />
+      <Topbar />
+
+      <div className="absolute top-1/2 left-1/2 z-10 flex w-screen -translate-x-1/2 -translate-y-1/2 flex-col justify-center">
```

Comment:
change `z-10` to `z-0` for resolving the signup modal and 404 error text overlapping